web/Controllers/AbstractController.cs

```diff
@@ -30,4 +30,18 @@ protected void FlashMessage(ILogger logger, (string logInfo, string flashtype, s
         TempData["FlashType"] = flash.flashtype;
         TempData["FlashMessage"] = flash.flashmessage;
     }
+    
+    protected void FlashMultipleMessages(ILogger logger, (List<string> logInfo, List<string> flashtypes, List<string> flashmessages) flashes)
+    {
+        var flashTypes = new List<string>();
+        var flashMessages = new List<string>();
+        for (int i = 0; i < flashes.logInfo.Count; i++)
+        {
+            logger.LogInformation(flashes.logInfo[i]);
+            flashTypes.Add(flashes.flashtypes[i]);
+            flashMessages.Add(flashes.flashmessages[i]);
+        }
+        TempData["MultipleFlashTypes"] = flashTypes;
+        TempData["MultipleFlashMessages"] = flashMessages;

```

Comment:
Would it be possible to just do `TempData["MultipleFlashTypes"] = flashes.flashtypes;` (and the same for `flashmessages`)? Or if you do need to copy it then maybe `TempData["MultipleFlashTypes"] = new List<string>(flashes.flashtypes)`.

Reply:
Having removed the logger from here, the entire code just got simpler so I can basically do that now (regardless of if would have worked before or not).